src/responsibleai/rai_analyse/_score_card/_rai_insight_data.py

```diff
@@ -283,6 +293,11 @@ def __init__(self, raiinsightdata, config):
             self.pos_label = str(self.data._classes[0])
             self.classes = [self.other_class, self.pos_label]
 
+    def validate_valid_metric_for_task_type(self):
+        for m in self.metrics:
+            if m in metric_task_map and metric_task_map[m].lower() != self.tasktype:
```

Comment:
Do we need to check for regression metrics?

Reply:
Within this module, we have supplemented some additional metrics mapping for metrics defined within the module, like the following:
metric_task_map["error_rate"] = "classification"

We have not had to supplement any regression metrics and they come from:
from erroranalysis._internal.metrics import metric_to_func, metric_to_task